api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -249,7 +288,7 @@ public void findCriteriaBy() {
   @Test
   public void findCriteriaByExceptions() {
     try {
-      controller.findCriteriaBy(1L, null, null, false, null);
+      controller.findCriteriaBy(WORKSPACE_NAMESPACE, WORKSPACE_ID, null, null, false, null);
       fail("Should have thrown a BadRequestException!");
     } catch (BadRequestException bre) {
       // success
```

Comment:
can remove the try catch block and use `@Test(expected = BadRequestException.class)`

Reply:
I think we would have to split this method in multiple (which is fine) since there are multiple try catch.

Reply:
Nvm i think i can use ExpectedException and use that